src/madpack/diff_udf.sql

```diff
@@ -11,71 +11,13 @@ RETURNS text AS $$
 $$ LANGUAGE plpythonu;
 
 
-CREATE OR REPLACE FUNCTION get_functions(schema_name text)
+CREATE OR REPLACE FUNCTION get_functions(table_name text, schema_name text,
+                                         type_filter text)
 RETURNS VOID AS
 $$
     import plpy
     plpy.execute("""
-        CREATE TABLE functions_madlib_new_version AS
-        SELECT
-            "schema", "name", filter_schema("retype", 'madlib') retype,
-            filter_schema("argtypes", 'madlib') argtypes, "type"
-        FROM
-        (
-
-            SELECT n.nspname as "schema",
```

Comment:
Why was this piece of code removed ?

Reply:
It was duplicated code. I just added another parameter to the function and reused it.